tensorboard/plugins/audio/audio_demo.py

```diff
@@ -18,18 +18,20 @@
 from __future__ import division
 from __future__ import print_function
 
+import inspect
 import math
 import os.path
 
 from six.moves import xrange  # pylint: disable=redefined-builtin
 import tensorflow as tf
+from tensorboard.plugins.audio import summary
 
 FLAGS = tf.flags.FLAGS
 
 tf.flags.DEFINE_string('logdir', '/tmp/audio_demo',
                        'Directory into which to write TensorBoard data.')
 
-tf.flags.DEFINE_integer('steps', 500,
+tf.flags.DEFINE_integer('steps', 50,
```

Comment:
(Note: I made this change because it cuts down the output from 617.8 MB to "only" 79.6 MB.)